lib/valueflow.cpp

```diff
@@ -7514,7 +7514,7 @@ static void valueFlowFunctionReturn(TokenList *tokenlist, ErrorLogger *errorLogg
         ProgramMemory programMemory;
         for (std::size_t i = 0; i < parvalues.size(); ++i) {
             const Variable * const arg = function->getArgumentVar(i);
-            if (!arg || !Token::Match(arg->typeStartToken(), "%type% %name% ,|)")) {
+            if (!arg) {
                 if (tokenlist->getSettings()->debugwarnings)
                     bailout(tokenlist, errorLogger, tok, "function return; unhandled argument type");
```

Comment:
We should probably do a selfcheck run with all the bailouts enabled and take a look at some of the non-frequent ones if those are legit. We could also split some of those off if it might be helpful so they show up. Or split of the very common ones instead.

Reply:
:+1:

Reply:
Okay - will take a look at that when my backlog gets more manageable.